packages/main/test/specs/Strings.spec.js

```diff
@@ -0,0 +1,23 @@
+const assert = require("chai").assert;
+
+describe("Strings utility", () => {
+  it("escapeRegex", () => {
+    assert.ok(new RegExp(escapeRegex(".*+?^${}()").test(".*+?^${}()")));
+  });
```

Comment:
I am not sure how to make the functions as defined; however, somehow magically the IDE finds them, so maybe this is fine.

Reply:
Although you crеated a pure unit test, we don’’t have a setup for such, and you have to create a html page (f.e. InputHighlighting.html or StringEscape.html), that the spec opens.

You can take use this spec.js as an example
https://github.com/SAP/ui5-webcomponents/blob/master/packages/base/test/specs/Theming.spec.js

especially the part
describe(“Theming works”, () => {
	before(async () => {
		**await browser.url("http://localhost:8080/test-resources/pages/AllTestElements.html");**
	}); (edited)

Reply:
@ilhan007 ,
I am really at a loss here as to exactly how does an HTML file test these Javascript files, or how do I actually run the test locally - since your build is a mess right now and doesn't function for me.

Can you or @MapTo0  take over, I have added access to you guys via the settings tab in my fork repository if you need to push changes.

Reply:
@MapTo0 can you take over and help adding the test, or you can add a test later on your own.